core/storage/boltdb/boltdbtest/db.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Package boltdbtest contains the utitilies needed for boltdb testing
+package boltdbtest
```

Comment:
Do we really need a dedicated package for test utils?

Reply:
I think it's a good practice. That way you end up reusing code instead of redefining it everywhere.